css-color-4/Overview.bs

```diff
@@ -4147,7 +4152,8 @@ Predefined Color Spaces</h2>
 
 		</dl>
 
-<h3 id="predefined-a98-rgb">Predefined a98-rgb space</h3>
+<h3 id="predefined-a98-rgb">
+The Predefined A* (1998) Color Space: the ''a98-rgb'' keyword</h3>
```

Comment:
Oh come on

Reply:
Having the actual keywords in the heading is helpful! We do it all over the place.

Reply:
He's complaining about the *.

Reply:
Ahaha ok lol